src/rn2xx3.h

```diff
@@ -12,6 +12,20 @@
 
 #include "Arduino.h"
 
+// Uncomment this line Use this to debug Library
```

Comment:
Why not do the same approach as the Sodaq_UBlox_GPS.h/cpp has, that has the methods setDiag() so that you can set a debug stream. This is used internally and that enabled you to enable debug/diagnostics from outside without having the need to recompile.